api/src/api.ts

```diff
@@ -152,6 +152,8 @@ export const plugin = {
     });
 
     /**
+     * This PUT route only lets you update an annotation; it does not let you create a new
+     * annotation. For the time being, the only way to create a new annotation is with POST.
```

Comment:
I think those are standard REST semantics, so `For the time being` can be omitted, as I think this is correct!

Reply:
Cool! I saw some examples online of REST servers that create on PUT when the ID is new, and [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT) also kind of hints that sometimes PUT creates a new object. That said, this sounds like optional behavior so I'll update the comment.